### PR TITLE
fix: Change stats path on win32

### LINKED ISSUE #31: Error: SSR requires `@loadable/babel-plugin`, please install it
When I run `gatsby build` I get the error 
```bash
WebpackError: Invariant Violation: loadable: SSR requires `@loadable/babel-plugin`, please install it
```
I'm on version `3.2.0` and the gatsby version is `3.7.1`.

The loadable component is 
```jsx
import Loadable from "@loadable/component"

const DistortedShader = Loadable(() => import("./DistortedShader"))

export {DistortedShader}
```

And then I'm importing it inside an `MDX` file

```jsx
import { DistortedShader } from "../../../demos"

<DistortedShader />
```

Not sure what's the problem though 😅

### PR DESCRIPTION
**Pull request by hazem3500**
Closes #31

### CODE REVIEW

**Comment by hazem3500** [comment]
It works on `build` but when I try to start development server it crashes 😅
Will check another solution

**Comment by hazem3500** [comment]
Changing the `outputAsset` to `true` made it work XD

File: src/gatsby-node.js [hunk 1]
```diff
@@ -2,13 +2,11 @@ import { unlinkSync } from "fs"
...
+const isWin = process.platform === "win32";
+
 const loadableStatsName = "loadable-stats-build-javascript.json";
-const statsPath = path.join(
-  process.cwd(),
-  `/.cache/${loadableStatsName}`
-)
 
-const isWin = process.platform === "win32";
+const statsPath = _path.default.join(process.cwd(), `/.cache/${loadableStatsName}`);
```
Comment (graysonhicks) [inline src/gatsby-node.js:9]: Where is `_path` defined?

File: src/gatsby-node.js [hunk 2]
```diff
@@ -19,8 +17,9 @@ export const onCreateWebpackConfig = ({ actions, stage }) => {
     actions.setWebpackConfig({
       plugins: [
         new LoadablePlugin({
-          filename: (isWin ? loadableStatsName : statsPath),
+          filename: statsPath,
           writeToDisk: true,
+          outputAsset: !isWin
```
Comment (graysonhicks) [inline src/gatsby-node.js:22]: Where does Windows write the file to?  I may want to include more details on Windows behavior in the docs.

File: src/gatsby-node.js (hunk 1 above)
Comment (hazem3500) [reply src/gatsby-node.js]: Oh my bad 😅, I was testing out the fix in `node_modules` where it has the `path` as `_path`, will change `_path.default` to `path`

File: src/gatsby-node.js (hunk 2 above)
Comment (hazem3500) [reply src/gatsby-node.js:22]: Before the fix it was writing in the root directory, now it should write it in the `.cache` directory.

The fix was taken from https://github.com/graysonhicks/gatsby-plugin-loadable-components-ssr/issues/25#issuecomment-865082674 workaround.

**Pull request merged**